src/avram/queryable.cr

```diff
@@ -243,6 +243,22 @@ module Avram::Queryable(T)
     end
   end
 
+  def group_count
```

Comment:
If that issue PGValue gets updated with the Ints, we can type this out to be `Hash(Array(PG::PGValue), Int64)`.

Reply:
```suggestion
  def group_count : Hash(Array(PG::PGValue | Int64), Int64)
```